openvdb_points_houdini/houdini/VRAY_OpenVDB_Points.cc

```diff
@@ -394,10 +518,52 @@ VRAY_OpenVDB_Points::render()
         gdp->addTuple(GA_STORE_REAL32, GA_ATTRIB_POINT, "pscale", 1, GA_Defaults(DEFAULT_PSCALE));
     }
 
+    // map speed to color if requested
+    if (mSpeedToColor) {
+        for (PointDataGridPtrVecIter    iter = mGridPtrs.begin(),
+                                        endIter = mGridPtrs.end(); iter != endIter; ++iter) {
+
+            PointDataGridPtr grid = *iter;
+
+            PointDataTree& tree = grid->tree();
+
+            PointDataTree::LeafIter leafIter = tree.beginLeaf();
+            if (!leafIter) continue;
+
+            if (leafIter->hasAttribute("Cd"))   dropAttribute(tree, "Cd");
```

Comment:
Why drop then re-add it? Perhaps better to only add it if it doesn't already exist (because it'll be edited anyway) ?